src/map/anticheat.h

```diff
@@ -0,0 +1,86 @@
+/*
+===========================================================================
+
+  Copyright (c) 2010-2019 Darkstar Dev Teams
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see http://www.gnu.org/licenses/
+
+  This file is part of DarkStar-server source code.
+
+===========================================================================
+*/
+
+#ifndef _CANTICHEAT_H
+#define _CANTICHEAT_H
+
+#include "../common/cbasetypes.h"
+#include "entities/charentity.h"
+
+namespace anticheat
+{
+    // Cheat type as appears in the cheat id column in the cheat_incidents table.
+    enum class CheatID : uint32
```

Comment:
Probably don't need to waste 4 bytes on this, if we have 4 billion cheats we need to track I'll just delete the repo instead.

Reply:
Fixed.